sdk/servicebus/tests.yml

```diff
@@ -7,4 +7,4 @@ extends:
     ServiceDirectory: servicebus
     SDKType: client
     TimeoutInMinutes: 120
-    Clouds: 'Public,Canary'
+    Clouds: 'Public,Preview,UsGov,China,Canary'
```

Comment:
If you're looking to also do this for the track one libraries, you'll need to reflect the change to tests.data.yml as well.